content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -311,6 +311,7 @@ kubectl run nginx --image=nginx                     # Run pod nginx and write it
 kubectl attach my-pod -i                            # Attach to Running Container
 kubectl port-forward my-pod 5000:6000               # Listen on port 5000 on the local machine and forward to port 6000 on my-pod
 kubectl exec my-pod -- ls /                         # Run command in existing pod (1 container case)
+kubectl exec --stdin --tty my-pod -- /bin/sh        # Interactive shell access to a running pod (1 container case) 
```

Comment:
Instead of using complete flag phrases(--stdin --tty), can we narrow it down using short forms?

Reply:
The command can look like `kubectl exec -it my-pod -- /bin/sh`

Reply:
In documentation, long forms are useful - they serve to explain things to the reader who may not understand abbreviations such as `-i` for connecting STDIN or `-t` for allocating a pseudoterminal.

Reply:
Got it! Thanks
/lgtm